src/interface/chessBoardInterface.js

```diff
@@ -1,5 +1,57 @@
-export default class ChessBoardInterface {
-    constructor(parentNode, onFieldClick) {
+// export default class ChessBoardInterface {
+//     constructor(parentNode, onFieldClick) {
 
-    }
-}
\ No newline at end of file
+//     }
+// }
+
+var symbols = {
+    "rookWhite" : "\u2656",
+    "knightWhite" : "\u2658",
+    "bishopWhite" : "\u2657",
+    "queenWhite" : "\u2655",
+    "kingWhite" : "\u2654",
+    "pawnWhite" : "\u2659",
+
+    "rookBlack" : "\u265c",
+    "knightBlack" : "\u265e",
+    "bishopBlack" : "\u265d",
+    "queenBlack" : "\u265b",
+    "kingBlack" : "\u265a",
+    "pawnBlack" : "\u265f",
+};
```

Comment:
Wprawdzie symbole mają swoje zalety, ale w ostatecznym rozrachunku rozważyłbym traktowanie wyświetlania na tej zasadzie albo jako fallbacka (kiedy z normalnym wyświetlaniem figur jest coś nie tak). Lepiej byłoby poszukać jakichś ładnych (darmowych ;) ) ikonek.

Reply:
in progress

Reply:
I closed, because this is not this task